activities/organization/finance-updates/finance-update-december-2020.md

```diff
@@ -0,0 +1,47 @@
+
+In October a decision was made to change the structure of the gift. On November 1st a transfer of € 1.726.000 was made to Public Code as a pre-payment of the last 4 terms of periodic gift dated 24-08-2020. The board of Public Code decided to use that money to repay the loan (€ 1.702.000) + interest (€ 9.652) to Dia Holding on November 1st. This brings an advantage of € 118.013 for Public Code.
+
+## General
+
+* The opening of a USD account is in progress and will help facilitate money transfers from KBFUS and to balance out the costs incurred by conversion rates and fees intermediary banks charge for international wire transfers.
+* The gift from Josh Elkes will be transferred as soon as this account is opened, which will be before the end of this year.
+
+## State of the administration
+
+* Budget sheet: the budget sheet is updated with the latest insights (new fiscal gift-loan of November, elimination of the amortization schemes from earlier gift-loan set-up, update of personnel cost as well as a realistic budget for 2021)
+* General ledger (Exact): Updated
+* VisionPlanner: Updated (interfaced with Exact)
+* WeFact: Set-up to facilitate membership administration (interfaced with Exact and Visionplanner)
+
+## Budget 2021 narrative
+
+The budget for 2021 is based on actuals and a fair estimation of additional costs we foresee for the upcoming year. We look 1 year ahead, as the Foundation is not a static organization and there are many unknown in terms of prospective members, potential new philanthropic gifts, as well as the impact the Covid crisis may have on travel for conferences and in-person community events. Most prominent changes to the budget are:
+
+* 2FTE added each for 5500 gross per month. Amortized over the year. Estimated starting date March for 1FTE and May for the other 1FTE. The need to hire more staff depends on the results of successful membership development
+* Salaries also include raises for some of the current staff
+* Office cost from Q2 for 6000 per month
+* Some budget for events and community from Q2
+* Marketing, research and design costs including illustration work by Floris Deerenberg and the design work by Ryan Conlan, of which the latter can be covered for the most part by the gift from Josh Elkes.
+* Development of a Jitsi Community user interface is included in the budget under this ledger as well. This interface is needed to enable our community to sign themselves up for our Jitsi Community instance.
+
+## Interim conclusion
+
+The new funding loan came in time for the continuation of funding for the Foundation. The runway is extended; the Opex (operational expenses) can be funded for the upcoming 19 months. Even though the runway is extended by AS, there is a need to get inflow of cash by:
```

Comment:
The reference to AS isn't self-explanatory. Would replacing it with a full name or a reference to a specific donation/loan make this more clear to a new reader?

Reply:
It would. It will differ from the original but any future original I will now make will take into account that it's written for to be out in the open and thus leaving abbreviations out of it